relnotes/queue-push-const.migration.md

```diff
@@ -16,9 +16,7 @@
   - `ubyte[] push ( size_t )` changed to `void[] push ( size_t )`
   - `bool push ( IUntypedQueue, void[] )` changed to
     `bool push ( IUntypedQueue, in void[] )`
-  - `ubyte[] pop ( )` changed to `Const!(void)[] pop ( )`
```

Comment:
`ubyte[] pop ( )` has still been changed, to `void[] pop ( )`.

Reply:
Updated